src/components/map-marker/map-marker.tsx

```diff
@@ -21,12 +19,24 @@ export class MapMarker implements GxComponent {
   @Prop({ mutable: true }) coords = "0, 0";
 
   /**
-   * The URL of the marker image.
+   * The class that the marker will have.
    *
-   * *Note: The image size must be 25 x 41*
+   * Set the `background-image` to use it as icon of the marker.
    *
    */
-  @Prop() readonly iconSrc: string;
+  @Prop() markerClass = "gx-defaultIcon";
```

Comment:
`gx-default-icon` should be the class name. Class names should be all lower case, hyphen separated.

Reply:
Fixed in 5bba66b